GVFS/GVFS.Common/Cleanup/PostFetchCleanupStep.cs

```diff
@@ -0,0 +1,47 @@
+﻿using GVFS.Common.Git;
+using GVFS.Common.Tracing;
+using System.Collections.Generic;
+using System.IO;
+
+namespace GVFS.Common.Cleanup
+{
+    public class PostFetchCleanupStep : GitCleanupStep

```

Comment:
Wondering if there should be a `WriteMultiPackIndexStep` and `WriteCommitGraphStep`  Are these two tied to each other?  Do they have to be run in order?  What if there are more post fetch steps?  Will they all be added to this class?

Reply:
1. They are currently run together, and this PR is intended to be a non-functional change when possible.
2. The event "I just ran prefetch" would trigger both of these events.
3. The commit-graph specifically needs the list of packfiles to run.
4. The commit-graph write will use the updated multi-pack-index file, but the multi-pack-index write does not use the commit-graph file.

So, in the future it _could_ make sense to queue these as separate cleanup jobs, but they would both be triggered by the same event. I'll leave that to a later change if we are interested in that.

Reply:
For us newbies it would be great if we put some comments at the top of the class saying what it Cleans up.  I.E. Those two steps above with a short reasoning.